### ISSUE TITLE
Individual page creation: field translation button is confusing

### ISSUE THREAD

**Issue by sepidehshahi** [accessibility-exchange/platform#728]

**Comment by greatislander** [comment]
"French translation of 'Field name'"

**Comment by cherylhjli** [comment]
Further discussion to use "Add [language] translation" - pending on Alan's feedback on whether this labelling provides enough context to a screenreader

**Comment by cherylhjli** [comment]
I have updated all the fields that require translation for the question to be short enough for us to use "French translation for ..." That being said, some of them required a bit more copy tweaking and we couldn't just use the field name.

Here's the proposed copy:

**Individual** 

Your bio: [Language] translation of your bio
Skills and strengths: [Language] translation of your skills and strengths
Lived experience: [Language] translation of your lived experience

**Community Organization**

About your organization: 
- [Language] translation of about your organization

Other areas of interest: 
- [Language] translation of your organization’s other areas of interest

**FRO**

About your organization: 
- [Language] translation of about your organization

**Project**

Please indicate the goals for this project: 
- [Language] translation of project goals

Please identify which communities this project will impact: - **EDIT: REMOVED THIS QUESTION**
- [Language] translation of the communities this project will impact

Please describe how the Disability and Deaf communities will be impacted by the outcomes of your project. **EDIT: Updated the phrasing of this question**
- [Language] translation of how communities will be impacted

Please provide an update about the work that has been completed since the previous project: 
- [Language] translation of project update

Please indicate what is “out of scope” for this project.
- [Language] translation for what is out of scope

Who will be going through the results from this project and making them into a report? **EDIT: UPDATED**
- [Language] translation of who is going through the results

Please indicate the tangible outcomes of this project.
- [Language] translation of tangible outcomes of this project 

Approximate response time
- [Language] translation of approximate response time

**Engagement**

What is the name of your engagement?
- [Language] translation of engagement name

Please describe this engagement.
- [Language] translation of engagement description

Title of meeting
- [Language] translation of meeting title

Interview - Accepted format - other
- [Language] translation of other accepted format

@greatislander @sepidehshahi please let me know what you think!

**Comment by cherylhjli** [comment]
Also, if the "[language] translation] part could be in bold, that would really help for sighted users to scan.

**Comment by cherylhjli** [comment]
@michelled made some updates to the project scope and project outcome questions based on copy related issues from accessibility testing. Please see the previous comment where I've labeled "EDIT" for changes.
